web-console/src/lib/components/streaming/management/PipelineTable.tsx

```diff
@@ -908,13 +899,13 @@ const PipelineActionsCell = (params: { row: Pipeline }) => {
           )}
           data-testid='button-delete'
         >
-          <IconTrashAlt fontSize={20} />
+          <i className={`bx bx-trash-alt`} style={{ fontSize: 24 }} />
         </IconButton>
       </Tooltip>
     ),
     spacer: () => (
       <IconButton size='small' sx={{ opacity: 0 }} disabled key='spacer'>
-        <IconStopCircle fontSize={20} />
+        <i className={`bx bx-stop-circle`} style={{ fontSize: 24 }} />
```

Comment:
should fontSize: 24 maybe be a default so you dont have to specify it almost everywhere?

Reply:
similarly the `bx ` in className

Reply:
maybe it should be in it's own component?

Reply:
I considered making a separate component to encapsulate raw <i/> tag but this wouldn't hide much complexity, and I'd rather have an inline indication of size rather than hide it behind a default - this is an approach I follow for styling in general